src/lib/logging.ts

```diff
@@ -0,0 +1,69 @@
+import Transport from "winston-transport";
+import { createDefaultTransportFormat } from "@zwave-js/core";
+import type { ZWaveLogInfo } from "@zwave-js/core";
+import { ClientsController, Logger } from "./server";
+import { Driver } from "zwave-js";
+
+export class LoggingEventForwarder {
+  /**
+   * Only load this once the driver is ready.
+   *
+   * @param clients
+   * @param driver
+   */
+  private serverTransport?: WebSocketLogTransport;
+  public started: boolean = false;
+
+  constructor(private clients: ClientsController, private driver: Driver) {
+    // Create log transport for server
+    this.serverTransport = new WebSocketLogTransport(this.clients);
+  }
+
+  start() {
+    if (!this.serverTransport || this.serverTransport === undefined) {
```

Comment:
The 1st check already covers teh 2nd check. Also, how can it be undefined if it's set in the constructor? You should remove the `!` from the definition .

Reply:
I swear I tried this before adding in this guard clause, but it's working now so 🤷🏾